docs/user-guide/data-structures/data-structures.ipynb

```diff
@@ -198,34 +154,20 @@
    "metadata": {},
    "source": [
     "An exception is raised from the `value` and `variance` properties if the variable is not 0-dimensional.\n",
-    "Note that a variable with one or more dimension extent(s) of 1 contains just a single value as well, but the `value` property will nevertheless raise an exception."
```

Comment:
We should not remove this. We recently even had a user question about this. Maybe move it to the note you added below?